docs/runtime/jsruntime-metrics.md

```diff
@@ -0,0 +1,192 @@
+|---|---|---|---|---|---|
+| [`jsruntime.eventloop.lag.type`](/docs/attributes-registry/jsruntime.md) | string | The type of the event loop latency. | `min`; `p90` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`jsruntime.version`](/docs/attributes-registry/jsruntime.md) | string | JS Runtime version. | `v22.0.0`; `v21.7.3` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+`jsruntime.eventloop.lag.type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `min` | Event loop minimum latency. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `max` | Event loop maximum latency. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `mean` | Event loop mean latency. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `stddev` | Event loop standard deviation latency. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `p50` | Event loop 50 percentile latency. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `p90` | Event loop 90 percentile latency. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `p99` | Event loop 99 percentile latency. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+### Metric: `jsruntime.eventloop.utilization`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.jsruntime.eventloop.utilization(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `jsruntime.eventloop.utilization` | Gauge | `s` | Event loop utilization. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+<!-- semconv metric.jsruntime.eventloop.utilization(full) -->
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`jsruntime.version`](/docs/attributes-registry/jsruntime.md) | string | JS Runtime version. | `v22.0.0`; `v21.7.3` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+### Metric: `jsruntime.gc.duration`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.jsruntime.gc.duration(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `jsruntime.gc.duration` | Gauge | `s` | Garbage collection duration. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
i think it's not a gauge it's a histogram

Reply:
I agree. Each collection is an entry in a histogram so you can view a summary of collection times over time

Reply:
I had this as a histogram, and because on the SIG meeting on May 1st, I understood that I cannot use histograms here, so I changed to gauge, maybe I misunderstood.

Reply:
check out [jvm.gc.duration](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/runtime/jvm-metrics.md#metric-jvmgcduration)

also, for any histograms, you'll want to specify the default buckets, check out https://github.com/open-telemetry/semantic-conventions/issues/316#issuecomment-1783528869

Reply:
Changed back to histogram and added info about bucket